src/main/java/com/epam/tc/App.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.tc;
+
+public class App {

```

Comment:
Why do you need this class?

Reply:
I just created this one for possible future tasks. Is it so bad?